src/pal/inc/pal.h

```diff
@@ -2570,6 +2570,9 @@ typedef struct _CONTEXT {
 
 #define CONTEXT_ALL (CONTEXT_CONTROL | CONTEXT_INTEGER | CONTEXT_SEGMENTS | CONTEXT_FLOATING_POINT | CONTEXT_DEBUG_REGISTERS)
 
+#define CONTEXT_XSTATE (CONTEXT_AMD64 | 0x40L)
+#define CONTEXT_XSTATE_YMMH (CONTEXT_AMD64 | 0x80L)
```

Comment:
It seems that we can use just CONTEXT_XSTATE like Windows do and not to introduce the CONTEXT_XSTATE_YMMH. I understand that we don't capture / restore the whole xstate, but I believe we will need to eventually add it for debugger purposes.